File: src/main/java/io/github/tanguygab/playerlistexpansion/sorting/HIGH_TO_LOW.java
package io.github.tanguygab.playerlistexpansion.sorting;

import org.bukkit.OfflinePlayer;

public class HIGH_TO_LOW extends SortingType {

    public HIGH_TO_LOW(String arg) {
        super(arg);
    }

    @Override
    public String getSortingString(String name, OfflinePlayer viewer) {
        String string = parse(name,viewer);
        double num = parseDouble(string);
        num = Integer.MAX_VALUE - num;
        return String.valueOf(num);
    }
}

File: src/main/java/io/github/tanguygab/playerlistexpansion/sorting/HIGH_TO_LOW.kt
package io.github.tanguygab.playerlistexpansion.sorting

import org.bukkit.OfflinePlayer

class HIGH_TO_LOW(arg: String?) : SortingType(arg) {
    override fun getSortingString(name: String, viewer: OfflinePlayer?): String {
        val string = parse(name, viewer)
        val num: Double = DEFAULT_NUMBER - parseDouble(string)
        return num.toString()
    }
}
